packages/firebase_auth/firebase_auth/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for Firebase Auth, enabling Android and iOS
   authentication using passwords, phone numbers and identity providers
   like Google, Facebook and Twitter.
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_auth/firebase_auth
-version: 0.15.1+1
+version: 0.15.2
```

Comment:
I think you can do this as a patch release (0.15.1+2) because there's no breaking change to the Dart API.

Pub should intelligently keep clients from upgrading to the new version if the pubspec.yaml changes aren't compatible.

/cc @amirh

Reply:
I think in semver aren't supposed to do "+" releases before 1.0.